### ISSUE TITLE
How to use the latest package versions with a Meteor track?

### ISSUE THREAD

**Issue by Sanjo** [meteor/meteor#5189]
Hi, we have a Velocity Meteor track for Velocity. When I use the `meteor test-packages --release velocity:METEOR@1.2-rc.15_1` command, the test application uses `mongo@1.1.1-server.perf.0` instead of `mongo@1.1.1-rc.4`. How do I get the same package versions for our Meteor track as the METEOR@1.2-rc.15 release? I merged the release tag into our branch already.

We have our own meteor-tool package (https://github.com/meteor-velocity/meteor/blob/velocity-release-1.2/packages/meteor-tool/package.js) and the release manifest is [this](https://github.com/meteor-velocity/meteor/blob/velocity-release-1.2/scripts/admin/meteor-release-velocity.json).

The release process I use is documented [here](https://velocity.readme.io/docs/publish-a-release#release-the-velocity-meteor-command-line-tool-fork).

**Comment by raix** [comment]
+1

**Comment by stubailo** [comment]
Hey, I asked @ekatek to give me some pointers on this, and here's what she came up with:

> Looks like what they are currently doing is publishing from manifest alone -- which basically means that their release contains ​*exactly*​ what they specify in their configuration file, which is the name of the tool package, but not anything else.

> So, if you are using their release, your versions of mongo, for example, are not pinned at all and can be whatever the constraint solver feels like.

> They need to fill out the manifest with the right packages. Luckily, they can use 'meteor show METEOR@1.2-rc.15 --ejson' to basically get the pre-completed ejson data.

Can you try that and see if it works?

**Comment by Sanjo** [comment]
Thanks for the fast response. I was able to release with pinned package versions. I ended up using `./meteor publish-release --from-checkout scripts/admin/meteor-release-velocity.json`. I also needed to change the hardcoded meteor-tool package name here: https://github.com/meteor-velocity/meteor/commit/48022bb4a782d6125fece897618dcae8695d981c. This way I don't need to update the package versions manually on each release.
